### DISCUSSION TITLE
可以自定义color type吗？

### DISCUSSION THREAD

**Discussion by HAM2506** [element-plus/element-plus#8063] [Q&A]
比如自定义一个粉色，通过`<el-button type="pink">Pink</el-button>`实现一个粉色按钮？

**Comment by tolking** [comment]
你可以查阅文档，里面有个 color API 满足你的需求

**Reply by HAM2506** [reply]
其实我的意思是自定一个type，使得不止button，包括message、badge或者其他有type属性的组件都适用于这个定义的type。

**Reply by tolking** [reply]
可扩展的 type 属性在目前并不好实现。因为要考虑到类型推导、全局的 type 配置、动态样式的生成等

建议可以通过自定义 CSS 通过 class 使用
